packages/contentstack-export/README.md

```diff
@@ -48,7 +48,7 @@ $ npm install -g @contentstack/cli-cm-export
 $ csdx COMMAND
 running command...
 $ csdx (--version)
-@contentstack/cli-cm-export/1.17.0 darwin-x64 node-v22.17.0
+@contentstack/cli-cm-export/1.17.0 darwin-arm64 node-v22.14.0
```

Comment:
If there is no change in the plugin, you don't need to push the README files too.